base_layer/core/src/transactions/key_manager/inner.rs

```diff
@@ -353,10 +353,7 @@ where TBackend: KeyManagerBackend<PublicKey> + 'static
 
                         // If we're trying to access any of the private keys, just say no bueno
                         if &TransactionKeyManagerBranch::Spend.get_branch_key() == branch {
-                            return wallet
-                                .private_spend_key
-                                .clone()
-                                .ok_or(KeyManagerServiceError::ImportedPrivateKeyInaccessible);
+                            return Ok(PrivateKey::default());
```

Comment:
I think this will have a conflict anyway, make sure to choose the current code in origin. This error here is fixed.